schema/cassandra/cadence/versioned/v0.24/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "0.24",
+  "MinCompatibleVersion": "0.23",
```

Comment:
0.24?

Reply:
This change is backwards compatible with 0.23 and so this is set to 0.23